shared/localization/format.js

```diff
@@ -5,11 +5,21 @@
  */
 'use strict';
 
+const fs = require('fs');
+
 const MessageFormat = require('intl-messageformat').default;
 const {isObjectOfUnknownValues, isObjectOrArrayOfUnknownValues} = require('../type-verifiers.js');
 
-/** Locales with messages. May be an empty object if bundled that way. */
-const LOCALES = require('./locales.js');
+/** Contains available locales with messages. May be an empty object if bundled. */
+const LOCALE_MESSAGES = require('./locales.js');
+
+/**
+ * The locale tags for the localized messages available to Lighthouse on disk.
+ * When bundled, these will be inlined by brfs.
+ */
+const DEFAULT_LOCALES = fs.readdirSync(__dirname + '/locales/')
+  .filter(basename => basename.endsWith('.json') && !basename.endsWith('.ctc.json'))
+  .map(locale => locale.replace('.json', ''));
```

Comment:
@connorjclark this idea came from your #10148, but rather than inserting it at build time, let's make it live in the file all the time. brfs will inline it (like it does with with [audit and gatherer paths](https://github.com/GoogleChrome/lighthouse/blob/524c62baee3a4a7d60fbdabe91ed18f33e12fe40/lighthouse-core/runner.js#L430-L441)), it'll fall back to the behavior you want if `locales.js` has been shimmed away, and it keeps the full locales list isolated to this file, rather than sharing more with the old `i18n.js`

Reply:
nice!